flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -1252,4 +1254,52 @@ void createFunction(
      * as one job.
      */
     StatementSet createStatementSet();
+
+    // FLIP-190 methods. They're considered experimental and might change in future versions
+
+    /**
+     * Load a plan starting from a {@link PlanReference} into a {@link CompiledPlan}. This will
+     * parse the input reference and will validate the plan.
+     *
+     * <p><b>Note:</b> This API is <b>experimental</b> and subject to change in future releases.
```

Comment:
Remove the additional warning in the JavaDocs. A warning in the website docs should be enough. In the end, we would like to encourage users to try it out.